arm-servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -2974,7 +2974,7 @@
     "EventHealthEvaluation": {
       "type": "object",
       "allOf": [{"$ref": "#/definitions/HealthEvaluation"}],
-      "x-ms-discriminator-value": "1",
+      "x-ms-discriminator-value": "EventHealthEvaluation",
```

Comment:
Yup. You should only have the extension for those that have a different name on the wire